debian/rules

```diff
@@ -51,6 +51,10 @@ override_dh_auto_install:
 
 override_dh_auto_build:
 
+	if ! pkg-config --modversion xfont2 1>/dev/null 2>/dev/null; then \
+	    sed -e 's/#define.*HasXfont2.*NO/#define HasXfont2 YES/' -i nx-X11/config/cf/host.def \
+	fi
```

Comment:
It actually is. There is no cleaner approach for building a Debian source package against various distro versions. I have added some clean-up stuff around it, but it basically needs to stay like this.